### ISSUE TITLE
Message logged when handling existing index in Elasticsearch is unclear about reason

### ISSUE THREAD

**Issue by fedde-s** [opentargets/platform#699]
Commit [7898b05](https://github.com/opentargets/data_pipeline/commit/7898b0513ff8ab6357394204f44b30696a2b2ad4#diff-6349ff0a45ba42bfe478f90d7068f53bR39) of the data pipeline, (included in the `master` branch via opentargets/data_pipeline#522) introduced a debug message about swallowing an ‘index exists exception’. This would happen when trying to create an index in Elasticsearch, either after Elasticsearch reported that the index did not exist, or after it has completed a call to the [subroutine to delete the index](https://elasticsearch-py.readthedocs.io/en/7.0.0/api.html#elasticsearch.client.IndicesClient.delete). 

Encountering this log message would leave me wondering why that is how the pipeline handles this circumstance, as my first impression would be that the index was _not_ actually successfully cleared.

**Comment by fedde-s** [comment]
@afaulconbridge, do you remember why this occurred? If you can't immediately think of a brief way to clarify the log message, I could try to rephrase your reply and make a pull request.

**Comment by fedde-s** [comment]
This affects pull request https://github.com/opentargets/data_pipeline/pull/517.

**Comment by afaulconbridge** [comment]
I think this will only happen when the index already exists (i.e. when the --val--append is used). With the changes in PR opentargets/data_pipeline#517 your right that this message should never be called.
I'm not sure what to do about it. For now, I suggest we leave it as it and you see if it comes up in your logs when using append.

**Comment by fedde-s** [comment]
I see. I'll look out for it once the code ends up in a release. If does turn out to be a state that should never occur at that point, I imagine it's best to let the exception bubble up and crash the pipeline. We'd have to think about why it occurred before we can decide how to the code should behave then.

**Comment by afaulconbridge** [comment]
closing, create a new issue if this continues
